docs/trusted-resources.md

```diff
@@ -14,7 +14,9 @@ weight: 312
 
 ## Overview
 
-Trusted Resources is a feature which can be used to sign Tekton Resources and verify them. Details of design can be found at [TEP--0091](https://github.com/tektoncd/community/blob/main/teps/0091-trusted-resources.md). This feature is under `alpha` version and support `v1beta1` version of `Task` and `Pipeline`.
+Trusted Resources is a feature which can be used to sign Tekton Resources and verify them. Details of design can be found at [TEP--0091](https://github.com/tektoncd/community/blob/main/teps/0091-trusted-resources.md). This is an alpha feature and supports `v1beta1` version of  `Task` and `Pipeline`.
+
+**Note**: trusted resources support verification of resources from OCI bundle or remote resolution, to use [cluster resolver](./cluster-resolver.md) make sure to set all default values for the resources before applied to cluster, otherwise the verification will fail due to the mutating webhook.
```

Comment:
I feel this paragraph is a bit confusing to users for 3 reasons: 1/. oci bundle is part of remote resolution  2. as discussed, there are 2 ways resolving cluster resources i.e. remote resolver, and local resolver. 3. what should users set for default values?

How about this?
**Note**: Currently, trusted resources only support verifying Tekton resources that come from remote places i.e. git, OCI registry and Artifact Hub. For in-cluster resources, verification might fail because the mutation webhook will set default values for the type of the parameters and results, which implies the resources will be modified before verification. In-cluster resources will be supported later. If one wants to use trusted resources for this case, they should set those default values for the resources before applying to the cluster so that mutation webhook doesn't affect the originality of the resource.



(Please add other areas that mutation webhook will take affect on if missed above)

Reply:
Emmm, OCI bundle is another feature not part of remote resolution, this feature is supported for v1beta1 but will not be supported in v1. I can update the doc to add link. https://github.com/tektoncd/pipeline/blob/main/docs/pipelineruns.md#tekton-bundles 

Thanks for the suggestion! I will update the doc.

Reply:
The link you shared is the remote resolver that resolves OCI resource. The [old bundle field in PipelineRef](https://github.com/tektoncd/pipeline/blob/main/docs/pipeline-api.md#pipelineref-1) will be deprecated for v1beta1. "Deprecated: Please use ResolverRef with the bundles resolver instead." 

Generally, I think we should mention where the resource came from (i.e. remote place not in-cluster), instead of mentioning remote resolution. 

> Thanks for the suggestion! I will update the doc. 

Thanks you for your hard work.